src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.CommandHelp;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.HelpCommand;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class UsageCommand extends MultiverseCoreCommand {
+    public UsageCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @HelpCommand
+    @Subcommand("help")
+    @CommandPermission("multiverse.core.help")
+    @CommandCompletion("@commands:mv")
+    @Syntax("[filter] [page]")
+    @Description("Show Multiverse-Core Command usage.")
+    public void onUsageCommand(CommandHelp help) {
+        if (help.getIssuer().isPlayer()) {
+            // Prevent flooding the chat
+            help.setPerPage(4);
```

Comment:
Maybe a `--all` flag to not show pages?

Reply:
hmm yea, possible

Reply:
looking at how CommandHelp context is parsed, I will hold off on this idea for now